Configuration/TypoScript/setup.typoscript

```diff
@@ -387,4 +387,4 @@ plugin.tx_form.settings.yamlConfigurations {
 
 module.tx_form.settings.yamlConfigurations {
     1543219101 = EXT:cart/Configuration/Yaml/FormSetup.yaml
-}
+}
```

Comment:
It seems you removed the newline at end of file.

Reply:
Done.